binding/Binding/SKObject.cs

```diff
@@ -71,7 +71,9 @@ protected override void DisposeManaged ()
 				}
 				dic.Clear ();
 			}
-			KeepAliveObjects?.Clear ();
+			if (keepAliveObjects is ConcurrentDictionary<IntPtr, SKObject> ka) {
+				ka.Clear ();
+			}
```

Comment:
Using the property will always construct a fairly expensive object. And we are doing it in the Dispose. This means that we are ALWAYS constructing this for EVERY object.